client/components/Checkbox/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Form from 'react-bootstrap/Form';
+
+const Checkbox = props => {
```

Comment:
I probably wouldn't have broken this out into it's on component, are you expecting that we will have a checkbox component that differs somehow from the underlying bootstrap checkbox component? I think it's possible we should just use the bootstrap checkbox directly if we are not modifying, for simplicity's sake.